src/formtest.cpp

```diff
@@ -126,15 +126,15 @@ int draw_group_doc() {
     auto gsid = gen.add_graphics_state(gs).value();
     {
         auto groupctx = gen.new_transparency_group(100, 100);
-        groupctx.cmd_w(10);
```

Comment:
All of these now leak memory, because the returned pointer is not `delete`d. Maybe something like:

```
std::unique_ptr<PdfDrawContext> groupctx{gen.new_transparency_group(100, 100)};
```